src/modules/launcher/Plugins/Microsoft.Plugin.Program/Storage/Win32ProgramRepository.cs

```diff
@@ -124,13 +124,20 @@ private void OnAppRenamed(object sender, RenamedEventArgs e)
             }
             catch (Exception ex)
             {
-                Log.Exception($"OnAppRenamed-{extension}Program|{oldPath}|Unable to create program from {oldPath}", ex, GetType());
+                Log.Exception($"OnAppRenamed-{extension} Program|{e.OldName}|Unable to create program from {oldPath}", ex, GetType());
```

Comment:
PT Run has several places that logs exception that are of very little interest.
Wouldn't be enough to log a simple warning without the full exception stack?

Reply:
It can be beneficial to have the type of exception and stack trace. In this case, we do not catch specific exception so it is hard to say what is expected to fail.

Reply:
Any exception that is logged must be fixed.
If the exception is just part of the regular workflow, it should only be logged as a warning.

Reply:
>If the exception is just part of the regular workflow, it should only be logged as a warning.

Agree with it. 
The above exception is not part of the regular workflow. If the exception is caught that means that `oldApp` is not initialized and it must not be the case for rename.